etc/lib/sys.js

```diff
@@ -38,6 +38,20 @@ function getDeviceInfo(frameworkName) {
   })
 }
 
+function getHardware(def_hardware) {
+  return new Promise((resolve, reject) => {
+    exec('uname -m', (err, stdout, stderr) => {
+      if (err) {
+        resolve(def_hardware)
+        return
+      }
+      
+      resolve(stdout.split('\n')[0] === 'aarch64' ? 'amlogic' : 
```

Comment:
in k18 dev board, command `uname -m` always return `arm`, then this expression would return `arm` but not `kamino`, currently we don't configured package for `arm`, so the boot service can't download and run `core` properly on k18

Reply:
So determine whether 'stdout.split('\n')[0]' is aarch64 or arm,
return `armlogic` if aarch64 and return`kamino` if arm, otherwise throw an `unsupported architecture` error

Reply:
Could you please list the all hardware types and I think one config for all hardwares is needed! 
e.g.:
```javascript
var HARDWARE = {
  amlogic: amlogic,
  aarch64: amlogic,
  kamino: kamino,
  arm: kamino,
  ...
}
```

Reply:
Done. Could please review it?